src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs

```diff
@@ -13,6 +14,29 @@ namespace Raven.Server.Documents.Handlers.Debugging
 {
     public class ThreadsHandler : RequestHandler
     {
+        [RavenAction("/admin/debug/remote-connections", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
+        public Task ListRemoteConnections()
+        {
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            using (var write = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                context.Write(write,
+                    new DynamicJsonValue
+                    {
+                        ["Remote Connections"] = new DynamicJsonArray(RemoteConnection.RemoteConnectionsList.ToList()
```

Comment:
This is debug endpoint, but we might want to process that via code, in which case the space in the name would be a stumbling block